baseapp/grpcserver.go

```diff
@@ -61,7 +61,9 @@ func (app *BaseApp) RegisterGRPCServer(server gogogrpc.Server) {
 		grpcCtx = context.WithValue(grpcCtx, sdk.SdkContextKey, sdkCtx)
 
 		md = metadata.Pairs(grpctypes.GRPCBlockHeightHeader, strconv.FormatInt(height, 10))
-		grpc.SetHeader(grpcCtx, md)
+		if err = grpc.SetHeader(grpcCtx, md); err != nil {
```

Comment:
There is practically no issue, but it would be better not to change the existing variable.
```suggestion
		if err := grpc.SetHeader(grpcCtx, md); err != nil {
```
It may also apply to other lines.